feedthefox/base/static/js/profile.js

```diff
@@ -7,4 +7,8 @@ jQuery(document).ready(function ($) {
         var term = t.val();
         (term !== '' ? $(target).show() : $(target).hide());
     });
+
+    if ($('#DeviceModal .errorlist').length > 0) {
```

Comment:
Because in order to show the modal you need to trigger ``modal('show')`` JS event.